scripts/sub-digest/subdigest.py

```diff
@@ -117,7 +117,8 @@ def _find_line_expr(self, expr):
 
     def _get_selection(self):
         if self.selection is None:
-            return self._get_section()
+            # return copy of section contents as list
+            return [line for line in self._get_section()]
```

Comment:
FYI, this is equivalent to
```suggestion
            return list(self._get_section())
```